docs/content/tutorial/saving-your-data.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: "Tutorial, part six: Saving your data | Dagster Docs"
+description: Learn how to use I/O managers to save your data.
+---
+
+# Tutorial, part six: Saving your data
+
+You've written a data pipeline using assets and had it rebuilt every hour by now. Currently, your assets are stored in a temporary folder that is deleted after closing the `dagster` process. **Note**: If you set up Dagster without the tutorial, you may have set a `DAGSTER_HOME` environment variable. If so, then your assets will be stored in that directory.
```

Comment:
Probably a nit, but ending the first sentence with `by now` feels odd to me. It doesn't technically change the meaning, but I read it as "the pipeline is rebuilt by now" and not "By now I've written a pipeline that's rebuilt"

Does that make sense? Feel free to ignore me if I'm grasping at straws 🥤